### DISCUSSION TITLE
After updating to v.4.46.0 can't receive emails anymore

### DISCUSSION THREAD

**Discussion by Tresillo2017** [simple-login/app#2146] [Self-hosting question] [answered]
If checked all my config, recreate all docker containers and read again the setup guide. I can send emails but nothings appears on the `sl-email` container. I've checked with `postmap` and everything works fine.

**Comment by Tresillo2017** [comment]
I'm getting this error when i try to send an email to any of my alias
```
Decision Engine classified the mail item was rejected because of IP Block (from outbound normal IP pools) -> 554 5.7.1 Service unavailable; Client host [40.107.223.123] blocked using zen.spamhaus.org; Error: open resolver; https://check.spamhaus.org/returnc/pub/172.68.133.5/
```

**Comment by Tresillo2017** [comment, marked as answer]
Error fixed by removing the following spam filters 
```less
reject_rbl_client zen.spamhaus.org,
reject_rbl_client bl.spamcop.net,
```
Thanks anyway!

**Reply by mrehanabbasi** [reply]
You can just remove the spamhaus one. That's the main culprit. Happened to me as well and I realized it after a whole month. Don't know who emailed me during that time. 😅
